crates/ty_project/src/metadata/options.rs

```diff
@@ -340,10 +340,11 @@ pub struct EnvironmentOptions {
     /// - `android` for Android
     /// - `ios` for iOS
     /// - `linux` for everything else
+    /// - `all` for all platforms
```

Comment:
This bulleted list describes the value ty will fall back to if no platform is specified. Currently we will never fall back to `all`, so I don't think it's correct to add `all` to this bulleted list. Instead, I think we should add a mention of `all` to the paragraph preceding this bulleted list